examples/layers/preprocessing/bounding_box/demo_utils.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Utility functions for preprocessing demos."""
+import matplotlib.pyplot as plt
+import numpy as np
+import tensorflow as tf
+import tensorflow_datasets as tfds
+
+from keras_cv import bounding_box
+
+
+def resize(image, label, img_size=(224, 224), num_classes=10):
+    image = tf.image.resize(image, img_size)
+    label = tf.one_hot(label, num_classes)
+    return {"images": image, "labels": label}
+
+
+def load_oxford_dataset(
```

Comment:
you can remove the oxford functions from the bounding box utils

Reply:
Done!